scram-tools.file/tools/cepgen/cepgen.xml

```diff
@@ -0,0 +1,22 @@
+<tool name="cepgen" version="@TOOL_VERSION@">
+  <info url="https://cepgen.hepforge.org/"/>
+  <lib name="CepGen"/>
+  <lib name="CepGenHepMC2"/>
+  <lib name="CepGenHepMC3"/>
+  <lib name="CepGenLHAPDF"/>
+  <lib name="CepGenProcesses"/>
+  <lib name="CepGenPythia6"/>
```

Comment:
there is ` libCepGenRoot.so`  too which links to `ROOTDataFrame` ( and all its deps). Is `CepGenRoot` required for any of the above libs?